common/lib/transport/connectionmanager.js

```diff
@@ -1,9 +1,8 @@
 var ConnectionManager = (function() {
-	var readCookie = (typeof(Cookie) !== 'undefined' && Cookie.read);
-	var createCookie = (typeof(Cookie) !== 'undefined' && Cookie.create);
-	var eraseCookie = (typeof(Cookie) !== 'undefined' && Cookie.erase);
-	var connectionKeyCookie = 'ably-connection-key';
-	var connectionSerialCookie = 'ably-connection-serial';
+	var getFromSession    = (typeof(SessionStorage) !== 'undefined' && SessionStorage.get);
```

Comment:
Sure, happy to do that. Though will still need to do `(typeof(SessionStorage) !== 'undefined' && ...` because the `sessionstorage.js` file isn't even loaded in node, so the `SessionStorage` object doesn't exist